alpha-lab/phone-neon/index.md

```diff
@@ -1,118 +1,103 @@
 
-Checking out the recording, Neon's accuracy richly captures gaze behaviour and provides a nice high-level overview of
-what the wearer was looking at.
+For a detailed explanation and visual representation of these metrics, you can refer to our [AOI tutorial](/gaze-metrics-in-aois/), which provides step-by-step instructions and charts for your analysis.
 
-Visualisations are great, but the real power of this tool is that it generates a CSV file containing gaze data mapped
-onto the screen, in 2D x, y coordinates. This offers many possibilities for further customisation and in-depth analysis.
+### Map Your Gaze Onto Dynamic Phone Screen Content
 
 <Youtube src="RKrf3YQjzao"/>
 
-## In the wild
+Until now, the tools that we have used have been limited to mapping gaze onto static images. However, the dynamic realm of mobile phones makes it important to study gaze behaviour in a more natural context, particularly during scrolling. We, therefore, used the [dynamic screen mapping tutorial](/map-your-gaze-to-a-2d-screen/) for this purpose. This approach provided a more comprehensive understanding of gaze behaviour in dynamic interfaces.
 
-That's all fine, but what does it look like to interact with phone screens when out and about? Consider this scenario:
-Imagine searching for products on your mobile app while looking at the physical products on a supermarket shelf. We
-brought Neon into the wild to assess its performance beyond controlled environments. See an example of real-world user
-behaviour below!
+Upon reviewing the recording, Neon demonstrated good accuracy in capturing gaze behaviour, offering a useful high-level overview of the wearer's focal points.
 
-<Youtube src="enkOC7_wf0U"/>
+Another very useful facet of this tool is its ability to generate a CSV file containing gaze data mapped onto the screen, represented in 2D x, y coordinates. This feature opens avenues for extensive customisation and more in-depth analysis, which we don’t cover here.
 
-## Let's wrap it up!
+## Let’s Wrap It Up!
 
-This article has unveiled the remarkable capability of Neon's calibration-free eye tracking to capture
-viewing behaviour during mobile app interactions - both inside and out and about. This tutorial, and its outcomes,
-are not limited to our specific use case and could be particularly useful for other types of UI/UX research. By combining
-Neon with the techniques we've highlighted here, you can gain invaluable insights into user engagement.
+Overall, we were impressed by Neon's capability to effectively characterise viewing behaviour on mobile apps, especially considering its calibration-free nature. The techniques outlined in this tutorial are not confined to our specific application; they hold significant value for various UI/UX and neuromarketing research endeavours. By integrating Neon with the methodologies discussed here, researchers can acquire invaluable insights into user engagement.
```

Comment:
Not sure if we need to be the ones to be "impressed". It is indeed impressive that we get such high accuracy, but the reader should be the one to come to that conclusion IMO.

Reply:
I agree - I modified the sentence: "In this article, we assessed whether Neon is capable of characterizing viewing behaviour on mobile apps. The techniques outlined in this tutorial are not confined to our specific application; they hold significant value for various UI/UX and neuromarketing research endeavours. By integrating Neon with the methodologies discussed here, researchers can acquire invaluable insights into user engagement."

Reply:
This works.